appengine/genetics/js/visualization.js

```diff
@@ -324,187 +535,218 @@ Genetics.Visualization.processCageEvents_ = function() {
   var getMouseName = Genetics.Visualization.getMouseName;
   while (Genetics.Cage.EVENTS.length) {
     var event = Genetics.Cage.EVENTS.shift();
+
+    var mouse = (event['ID'] != null) ?
+        Genetics.Visualization.mice_[event['ID']] : null;
```

Comment:
Can be simplified to:
var mouse = event['ID'] && Genetics.Visualization.mice_[event['ID']];
Here, and next two lines.